Library/Homebrew/shims/super/cc

```diff
@@ -470,7 +470,7 @@ if __FILE__ == $PROGRAM_NAME
 
   args << { close_others: false }
   if mac?
-    exec "#{dirname}/xcrun", tool, *args
+    exec({"DEVELOPER_DIR" => nil}, "/usr/bin/#{tool}", *args)
```

Comment:
Won't this break formulae that compile with GCC or LLVM Clang on macOS?

Reply:
Do you have an example? Spot-checking with source builds of adios2 and libatomicparsley (both of which have a build dependency on gcc) succeeded.

Reply:
They don't have a `fails_with :clang`, though. Try `openblas` or `fftw`.

Reply:
Both `openblas` and `fftw` failed, as you expected.

Reply:
Yes, this will need fixing. If you have ideas for it feel free to open a new PR branching off of this one.

Reply:
I can change this to `tool` without the path, but we'd need to check it doesn't mix up clang between system and, say, the LLVM formula.

Reply:
It'll probably also keep calling itself if it's just `clang`, no?

Reply:
The Linux part a few lines done has logic to avoid this.

Reply:
It should be fine then, since for `llvm_clang` we have `tool = #{HOMEBREW_OPT}/llvm/bin/clang`, right?

Reply:
Can we just do
```ruby
if mac? && tool.match?(/^clang(\+\+)$/)
  exec "/usr/bin/xcrun", tool, *args
else
  ...
```
?